test/models/dosage_test.rb

```diff
@@ -67,7 +67,7 @@ class DosageTest < ActiveSupport::TestCase
       deny bad_dosage.valid?
       
       # destroy the visit by Dusty
-      @visit_dusty.destroy    
+      @visit_dusty.delete
```

Comment:
bc of the new foreign key restrictions. ideally test cleanup would not have to be handled manually by us tho 🤔